test/config/TestConfig.cpp

```diff
@@ -1258,6 +1258,9 @@ TEST_F(ConfigTestFixture, SensorPublishInvalidConfigAddr)
     PlainConfig config;
     config.LoadFromJson(jsonView);
 
+#if !defined(EXCLUDE_SENSOR_PUBLISH)
```

Comment:
shouldn't this be like this without negation `!`?

```
#if defined(EXCLUDE_SENSOR_PUBLISH)
    GTEST_SKIP();
#endif
```

Reply:
Dooh! Thank you @HarshGandhi-AWS